### ISSUE TITLE
Changing things in app settings doesn't actually change it (version for example)

### ISSUE THREAD

**Issue by dallinac** [pebble/cloudpebble#144]
When changing things such as the version number, and the app icon, it doesn't actually change them when compiled. The only way I could find around this was changing the pbw to a zip and then editing the appinfo.json, then changing it back to a pbw. This is a serious issue that needs to be fixed.

**Comment by Katharine** [comment]
Did you click the "save" button at the bottom of the settings pane?

**Comment by dallinac** [comment]
Yes. I even tried logging out and back in as well as refreshing the page among other things. The main thing is that I cannot upload a newer version of my app since the app version will not save. This also has happened to my friend who was trying to submit a newer version of his app.
